docs/topics/gradle.md

```diff
@@ -927,12 +927,12 @@ Here is a complete list of options for Gradle tasks:
 
 ### Attributes common to JVM, JS, and JS DCE
 
-| Name | Description | Possible values |Default value |
-|------|-------------|-----------------|--------------|
-| `allWarningsAsErrors` | Report an error if there are any warnings |  | false |
-| `suppressWarnings` | Don't generate warnings |  | false |
-| `verbose` | Enable verbose logging output. Works only when the [Gradle debug log level enabled](https://docs.gradle.org/current/userguide/logging.html) |  | false |
-| `freeCompilerArgs` | A list of additional compiler arguments |  | [] |
+| Name | Description                                                                                                                                         | Possible values |Default value |
+|------|-----------------------------------------------------------------------------------------------------------------------------------------------------|-----------------|--------------|
+| `allWarningsAsErrors` | Report an error if there are any warnings                                                                                                           |  | false |
+| `suppressWarnings` | Don't generate warnings                                                                                                                             |  | false |
+| `verbose` | Enable verbose logging output. Works only when the [Gradle debug log level enabled](https://docs.gradle.org/current/userguide/logging.html)         |  | false |
+| `freeCompilerArgs` | A list of additional compiler arguments. You can use experimental `-X` arguments here too. See an [example](#example-of-additional-arguments-usage) |  | [] |
```

Comment:
Is it possible somehow to mention that `freeCompilerArgs` usage is discouraged. If user is missing some option in Gradle DSL - new Kotlin issue should be opened?

Reply:
Sure